### ISSUE TITLE
Move mp3/ogg lobby music into cacheable asset bundle

### ISSUE THREAD

**Issue by fomalsd** [unitystation/unitystation#4236]
Each time you download a client update you have to redownload same 50-60mb of lobby music bundled with the build.
Would be cool if music could be separated into an asset bundle and downloaded only once (and perhaps updated if more tracks are added, but that's not a priority).

Tracker files (`.mod`,`.xm`,`.sunvox`) are tiny and ok to be bundled with each build in my opinion.

This might require a hub update.
